combine-trend-data.py

```diff
@@ -46,12 +48,24 @@ def combine_dicts(tsv_dict_1, tsv_dict_2):
 
     return tsv_dict_1
 
-def dict_to_tsv(combined_dict, output_tsv_path):
+#this assumes that the first column of each header is the same primary key
+def combine_headers(headers_1, headers_2):
+    print("Combining :" + str(headers_1))
+    print("With :" + str(headers_2))
+    if headers_1[0] != headers_2[0]:
+        exit("First column of the headers do not match. Aborting!")
+
+    combined_headers = []
+    headers_2_without_primary_key = headers_2[1:]
+
+    combined_headers = headers_1 + headers_2_without_primary_key
+    print("combined_headers list: " + str(combined_headers))
+    return combined_headers
+
+def dict_to_tsv(combined_dict, combined_headers, output_tsv_path):
     with open(output_tsv_path, 'wt') as out_file:
         tsv_writer = csv.writer(out_file, delimiter='\t')
-        headers = ["gene_symbol", "chromosome", "start", "length", "color", "full_name", "days_in_timeframe", "recent_timeframe_citation_count", "past_timeframe_citation_count", "citation_count_delta", "significance", "wikipedia_daily_page_views", "wikipedia_daily_page_views_change_from_previous_day"]
```

Comment:
basically the whole point of this PR is to reduce the need to keep updating this random line every time we add new columns to either the wiki tsv or the citations tsv